### PR TITLE
feat: Add vector2.ToSize Extension

### LINKED ISSUE #4860: Add Rect.ActualSize.ToSize() support
## What would you like to be added:

Add `Rect.ActualSize.ToSize()` support where ActualSize returns a `Vector2`

## Why is this needed:
Being able to translate a Rect `Vector2` to `Size`

## For which Platform:

- [x] iOS
- [x] Android
- [x] WebAssembly
- [ ] WebAssembly renderers for Xamarin.Forms
- [x] macOS
- [x] Skia
  - [x] WPF
  - [x] GTK (Linux)
  - [x] Tizen
- [ ] Windows
- [ ] Build tasks
- [ ] Solution Templates

### PR DESCRIPTION
**Pull request by ajpinedam**
GitHub Issue (If applicable): #
closes #4860 
 

## PR Type

What kind of change does this PR introduce?

- Feature

## What is the current behavior?
No support for Vector2.ToSize()

## What is the new behavior?
Uno.Extension to support for Vector2.ToSize()

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [x] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information
This extension is part of the System.Numerics.Vectors.WindowsRuntime nuget =>  https://www.nuget.org/packages/System.Numerics.Vectors.WindowsRuntime/

Unfortunately, this one is only available for the target uap

Internal Issue (If applicable):

### CODE REVIEW

**Review by davidjohnoliver — APPROVED** [review]

**Review by MartinZikmund — APPROVED** [review]

File: src/Uno.UI/Extensions/VectorExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Numerics;
+using System.Text;
+using Windows.Foundation;
+
+namespace Uno.UI.Extensions
+{
+	public static class VectorExtensions
```
Comment (jeromelaban) [inline src/Uno.UI/Extensions/VectorExtensions.cs:9]: Are those extensions on a `Windows.*` namespace in WinUI ?

Comment (ajpinedam) [reply src/Uno.UI/Extensions/VectorExtensions.cs:9]: In UWP these extensions come from the [nuget package](https://www.nuget.org/packages/System.Numerics.Vectors.WindowsRuntime/) `System.Numerics.Vectors.WindowsRuntime`

Comment (ajpinedam) [reply src/Uno.UI/Extensions/VectorExtensions.cs:9]: > Are those extensions on a `Windows.*` namespace in WinUI ?

In UWP these extensions come from the [nuget package](https://www.nuget.org/packages/System.Numerics.Vectors.WindowsRuntime/) `System.Numerics.Vectors.WindowsRuntime`

Comment (jeromelaban) [reply src/Uno.UI/Extensions/VectorExtensions.cs:9]: We probably should be using the same namespace for the extension class.

Comment (ajpinedam) [reply src/Uno.UI/Extensions/VectorExtensions.cs:9]: @jeromelaban  updated the namespace to match the one in the original package

**Review by jeromelaban — APPROVED** [review]

**Pull request merged**